common/flatpak-dir.c

```diff
@@ -6790,7 +6788,6 @@ export_desktop_file (const char         *app,
       old_exec = g_key_file_get_string (keyfile, groups[i], "Exec", NULL);
       if (old_exec && g_shell_parse_argv (old_exec, &old_argc, &old_argv, NULL) && old_argc >= 1)
         {
-          int i;
```

Comment:
This isn't right, we already iterate over in in the containing groups[] loop, so we need to use a different loop variable here. Say `j`

Reply:
Good catch